laravel-app/app/Services/PostService.php

```diff
@@ -10,10 +10,8 @@ class PostService implements PostServiceInterface
 {
     /**
      * To create a new post.
-     * @param User $user
-     * @param array<mixed, mixed> $validatedData
      */
-    public function createPost(User $user, array $validatedData): UserPost
+    public function createPost(User $user, array $validatedData): UserPost // @phpstan-ignore-line
```

Comment:
why ignore?

Reply:
sir the one you commented is the outdated version of the PR, I already removed them from last 2 commits. Thank you